<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassicWindow</class>
 <widget class="QMainWindow" name="ClassicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>405</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MoonPlayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout" stretch="1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="playerLayout">
      <property name="spacing">
       <number>6</number>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="toolbar" native="true">
      <layout class="QHBoxLayout" name="toolbarLayout" stretch="0,0,0,0,1,0,0,0,0">
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Play</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/classic_ui/play.png</normaloff>:/classic_ui/play.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Pause</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/classic_ui/pause.png</normaloff>:/classic_ui/pause.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Stop</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/classic_ui/stop.png</normaloff>:/classic_ui/stop.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="timeSlider">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="durationLabel">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="volumeButton">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Volume</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/classic_ui/volume.png</normaloff>:/classic_ui/volume.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="netButton">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Explore</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/classic_ui/net.png</normaloff>:/classic_ui/net.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playlistButton">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Playlist</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/classic_ui/playlist.png</normaloff>:/classic_ui/playlist.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open(&amp;O)</string>
    </property>
    <addaction name="actionAdd_file_s"/>
    <addaction name="actionAdd_url"/>
    <addaction name="actionAdd_playlist"/>
   </widget>
   <widget class="QMenu" name="menuTools_T">
    <property name="title">
     <string>Tools(&amp;T)</string>
    </property>
    <addaction name="actionOnline_videos"/>
    <addaction name="separator"/>
    <addaction name="actionCut_video"/>
    <addaction name="actionScreenshot"/>
    <addaction name="separator"/>
    <addaction name="actionUpdate_plugins"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuAbout_A">
    <property name="title">
     <string>About(&amp;A)</string>
    </property>
    <addaction name="actionBrowser_extension"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuTools_T"/>
   <addaction name="menuAbout_A"/>
  </widget>
  <action name="actionAdd_file_s">
   <property name="text">
    <string>Add file(s)</string>
   </property>
  </action>
  <action name="actionAdd_url">
   <property name="text">
    <string>Add url</string>
   </property>
  </action>
  <action name="actionAdd_playlist">
   <property name="text">
    <string>Add playlist</string>
   </property>
  </action>
  <action name="actionOnline_videos">
   <property name="text">
    <string>Online videos</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionBrowser_extension">
   <property name="text">
    <string>Browser extension</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdate_plugins">
   <property name="text">
    <string>Update plugins</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionCut_video">
   <property name="text">
    <string>Cut video</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
